### ISSUE TITLE
Accessibility - Sectors Screen - Not Reading List Contents

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Local-Labor-Market-Data-Android#11]
When using TalkBack to read the content on the sectors screen after reading the table headings, all it says is "List" after that and does not read the information in the table.

See screenshot to show content not being read. For some reason if I get a few screen deep it then reads the table contents.

![bls - android - sectors screen - tablkback not reading list contents](https://user-images.githubusercontent.com/10423487/56764448-b610d580-6772-11e9-9a41-50c1b96b79d5.png)

**Comment by greg-gruse** [comment]
Are you using a specific gesture to tab through the table headings and then into the list?   If so, could you let me know the gesture you are using?  

When I get to this screen it does allow me to tap into the table cells - now when it reads them it doesn't read all of the labels one after another but basically just each label within the table cell that you tap on.   So, for example "Government (9000000)" is read by itself.

**Comment by jfc3-dol** [comment]
I believe I started with doing an "L" and then selecting "Read from Top" or "Read Next Item" and then when that didn't work even attempted one finger left to right swipe to read one item at a time, and it would go an read the TABLE contents.

I was able to read some of it but not the button at the end of the row if I tapped in the TABLE, but a user that is using TalkBack is most likely not sighted to know there is more content after the TABLE headings as I am.

I can attempt again on our devices if this doesn't work for you and give you more detail if necessary.

**Comment by greg-gruse** [comment]
Ok, I see what you mean now just swiping through the items in Talkback,  I think I had my accessibility next feature set to the wrong type.   I will investigate more.

**Comment by greg-gruse** [comment]
I fixed it so that you can tab into the table / list.   The only thing is it wants to read the entire list item with all of its fields.  Is that ok?  I can craft the talk back message to read more as a sentence if that would help.

**Comment by greg-gruse** [comment]
This should be fixed in the  May 8th build?

**Comment by jfc3-dol** [comment]
Thanks, for the fix but TalkBack is now reading the entire row of content. If I want to go back to read something on a line because I didn't hear it, I have to listen to the entire line instead of only one item at a time.

The first highlighted item reads the whole row. The individually highlighted and numbered items are the order I would expect TalkBack to read them in.

Please see the attached screenshot.

![bls - android - detail city screen - expected row reading order](https://user-images.githubusercontent.com/10423487/57400603-de042e00-71a1-11e9-971d-b1061c9ff9e3.png)

**Comment by greg-gruse** [comment]
Fixed in the next build

**Comment by jfc3-dol** [comment]
Thanks, for making TalkBack read the TABLE row items as individual items instead of one row.

But TalkBack is not reading the button/link (arrow) at the end, so people know that there is more detailed information for that row/record.

**Comment by jfc3-dol** [comment]
It's now reading the right-facing arrow as a button that reads "More details button double tap to activate" Which is a great start. Can you add the text from the first column in front of that, so the screen reader user will know which button it is? Don't need the numbers if present only the words.

**Comment by jfc3-dol** [comment]
Thanks, TalkBack is now reading the content from the first column before telling the user that it's a more detail button for the Metro, State, and County detail screens. 

I wasn't expecting the numbers to be read as part of the button, but if that is how it is, I will close the issue.

**Comment by jfc3-dol** [comment]
TalkBack is not reading the numbers along with the text from the first column as part of the more details button.

I am closing this issue since it's been corrected.
